composer.json

```diff
@@ -40,7 +40,7 @@
         "symfony/filesystem": "^5.0 || ^6.0",
         "symfony/finder": "^4.4 || ^5.0 || ^6.0",
         "symfony/http-kernel": "^4.4 || ^5.0 || ^6.0",
-        "symfony/phpunit-bridge": "^5.0"
+        "symfony/phpunit-bridge": "^5.3 || ^6.0"
```

Comment:
I don't think we need to support 6.0.

For the same reason we use fixes versions for phpstan and psalm, we should use only one version for phpunit

Reply:
Sure. You are correct